src/openforms/js/components/admin/form_design/variables/VariablesTable.js

```diff
@@ -0,0 +1,129 @@
+import React, {useContext, useState} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+import useAsync from 'react-use/esm/useAsync';
+
+import {Checkbox, TextInput} from '../../forms/Inputs';
+import Select from '../../forms/Select';
+import {DATATYPES_CHOICES} from './constants';
+import DeleteIcon from '../../DeleteIcon';
+import {PluginsContext} from '../Context';
+import {get} from '../../../../utils/fetch';
+import FAIcon from '../../FAIcon';
+
+
+const SensitiveData = ({isSensitive}) => {
+    const intl = useIntl();
+    const sensitiveTitle = intl.formatMessage({
+        description: 'Is sensitive icon title',
+        defaultMessage: 'Is sensitive',
+    });
+    const notSensitiveTitle = intl.formatMessage({
+        description: 'Is not sensitive icon title',
+        defaultMessage: 'Is not sensitive',
+    });
+
+    return (
+        isSensitive
+        ? <FAIcon icon="check" title={sensitiveTitle} />
+        : <FAIcon icon="times" title={notSensitiveTitle} />
```

Comment:
we might want to correlate "not sensitive" with a success icon and "sensitive" with a danger icon for more accurate sentiment representation.

Reply:
I was thinking maybe the standard admin way of representing the `bool` is also nice (but then the red is for 'not sensitive' => so I'm not sure if it will be confusing then to swap them around?)

![Screenshot from 2022-05-20 09-36-17](https://user-images.githubusercontent.com/19154114/169477816-5999fe40-9e4e-4242-8381-eeb1ef3e09ba.png)
![Screenshot from 2022-05-20 09-36-32](https://user-images.githubusercontent.com/19154114/169477821-6c6b0453-ccaa-422d-8c4b-a16a0919c4f4.png)

Reply:
that's exactly the sentiment I mean, and you can work around it by inverting the label, something like "not sensitive data". In this case however, it feels forced, so I still prefer the ok/danger icon approach.